yarn.lock

```diff
@@ -2767,14 +2767,14 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@playwright/test@npm:^1.20.2":
-  version: 1.41.1
-  resolution: "@playwright/test@npm:1.41.1"
+"@playwright/test@npm:^1.41.2":
```

Comment:
I'm not sure why this update was necessary, could you explain?

The linked issue relates to a change fixed in 1.24 and this repo was already using 1.41.
Updating the version in the package.json forces others to update, which would make this PR a bigger change than it needs to be.